src/docs/product/sentry-basics/search/saved-searches.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Saved Searches
+sidebar_order: 20
+keywords: ["saved search", "premade searches"]
+description: "Learn more about recommended, pinned, and saved searches."
+---
+
+In the "Saved Searches" tab of **Issues**, you can access recommended searches, pin searches that you find most useful, and take advantage of organization-wide saved searches.
+
+## Recommended Searches
+
+Recommended searches are common search terms that we think you're likely to use. These premade searches are listed under "Recommended Searches" in the "Saved Searches" dropdown, and they're listed in order of when you've most recently used them.
+
+![Open Saved Searches dropdown on the Issues page.](search-premade-dropdown.png)
+
+## Pinned Searches
+
+You can pin a search, and it will become the default view you see on the **Issues** page. The pinned search is only visible to you and is relevant across your projects.
```

Comment:
```suggestion
You can pin a search, and it will become the default view you see on the **Issues** page. The pinned search is visible only to you and is relevant across your projects.
```